src/features/memoroom/memoRoomSlice.js

```diff
@@ -42,6 +42,30 @@ export const slice = createSlice({
       state.isLoading = false;
       state.error = action.payload;
     },
+    addNewMemoRequest: (state) => {
+      state.isLoading = true;
+    },
+    addNewMemoSuccess: (state, action) => {
+      const { newMemo } = action.payload;
+
+      state.memos = {
+        ...state.memos,
+        [newMemo._id]: {
+          formType: newMemo.formType,
+          content: newMemo.content,
+          location: newMemo.location,
+          size: newMemo.size,
+          color: newMemo.color,
+          alarmDate: newMemo.alarmDate,
+          tags: newMemo.tags,
+        },
+      };
```

Comment:
```suggestion
      state.memos[newMemo._id]: {
          formType: newMemo.formType,
          content: newMemo.content,
          location: newMemo.location,
          size: newMemo.size,
          color: newMemo.color,
          alarmDate: newMemo.alarmDate,
          tags: newMemo.tags,
        };
```
리덕스 툴킷을 사용하니 해당 툴킷에는 immer 내장되어있어 위 처럼 사용할 수 있을것 같은데 어떠신가요??

Reply:
아 immer가 내장되어있군요 ! 제안 주신 방식으로 리팩토링 하겠습니다.

Reply:
넵 알겠습니다!!